content/millennium/r4/scheduling/appointment.md

```diff
@@ -66,23 +66,25 @@ _Implementation Notes_
+-----------------------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------
+ `_id`                 | Yes, or one of `patient`, `practitioner`, or `location`.                           | [`token`]     | The logical resource id associated with the Appointment. Example: `3005759`
+ `date`                | Yes, or `-date-or-req-period` when using `patient`, `practitioner`, or `location`. | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
+ `-date-or-req-period` | Yes, or `date` when using `patient`, `practitioner`, or `location`.                | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
+ `patient`             | Yes, or `_id`                                                                      | [`reference`] | A single or comma separated list of Patient references. Example: `4704007`
+ `practitioner`        | Yes, or `_id`                                                                      | [`reference`] | A single or comma separated list of Practitioner references. Example: `2578010`
+ `location`            | Yes, or `_id`                                                                      | [`reference`] | A single or comma separated list of Location references. Example: `633867`
+ `status`              | No                                                                                 | [`token`]     | A single or comma separated list of appointment statuses. Example: `arrived`
+ [`_count`]            | No                                                                                 | [`number`]    | The maximum number of results to return.
 
 Notes:
 
 * The `patient`, `practitioner`, and `location` parameters may be included only once and may not be used in combination. For example, `patient=1234,5678` is supported but `patient=1234&patient=5678` and `patient=1234&location=5678` are not.
-* The `date` parameter may be provided:
+* Either `date` or `-date-or-req-period` parameter may be provided:
   * once with a prefix and time component to indicate a specific date/time. (e.g. `&date=ge2019-12-07T22:22:16.270Z`, `&date=lt2019-12-14T22:22:16.270Z`)
   * twice with the prefixes `ge` and `lt` to indicate a specific range. The date and prefix pairs must define an upper and lower bound. (e.g. `&date=ge2019-12-07T22:22:16.270Z&date=lt2019-12-14T22:22:16.270Z`)
 * The retrieved appointments are sorted first by `start` date ascending (earliest first), followed by the provided search parameter (`patient`, `practitioner` or `location`) and `start` time ascending (earliest first).
+* Search by `date` returns only non-proposed appointments whereas search by `-date-or-req-period` returns both proposed and non-proposed appointments.
```

Comment:
this sentence doesn't make sense, because we haven't previously defined what a non-proposed appointment is. We should make a note talking about which fields the parameter is using to search, and I think that should be enough to describe the search parameter.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/72ae8eac60f6180eb4fdf81cf6184e39a2b326eb#diff-1d7df8ceb9f0ae8471323516e037c662R87